src/tests/JIT/Regression/JitBlue/Runtime_105474/Runtime_105474.csproj

```diff
@@ -0,0 +1,8 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Optimize>True</Optimize>
+  </PropertyGroup>
```

Comment:
Should we add `<CLRTestTargetUnsupported Condition="'$(TargetArchitecture)' != 'arm64'">true</CLRTestTargetUnsupported>` to avoid building this test unnecessarily?